.ruby-version

```diff
@@ -1 +1 @@
-3.3.0
+3.3.5
```

Comment:
Was upgrading to 3.3.5 required as part of the Rails upgrade?

Reply:
It wasn't but after I went to 3.3.0, I realized .5 was out and thought it was worthwile to update to latest, including fixing some memory leaks (which we shouldn't be impacted by but still better safe than sorry)!